Gems/LyShine/Code/Editor/LyShineEditorSettings.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include "AzCore/base.h"
+#include "ViewportInteraction.h"
+
+namespace LyShine
+{
+    static constexpr AZStd::string_view InteractionModeSetting = "/O3DE/Preferences/Editor/LyShine/InteractionMode";
+    static constexpr AZStd::string_view CoordinateSystemSetting = "/O3DE/Preferences/Editor/LyShine/CoordinateSystem";
```

Comment:
update: I'd suggest moving this to the .cpp file so they don't leak out and people use the functions instead (also it's best not to use `static` when defining variables in the .h, instead use `inline` or move the definition to the .cpp).